src/documentation.jl

```diff
@@ -186,16 +186,16 @@ SetRingDim
 """
     GetPlaintextModulus(parameters::EncodingParams)
 
-FIXME!
+Return the plaintext modulus used by the given `parameters`.
 
 See also: [`EncodingParams`](@ref), [`SetPlaintextModulus`](@ref)
 """
 GetPlaintextModulus
 
 """
-    SetPlaintextModulus(parameters::EncodingParams, FIXME::Integer)
+    SetPlaintextModulus(parameters::EncodingParams, plaintextModulus::Integer)
```

Comment:
```suggestion
    SetPlaintextModulus(parameters::EncodingParams, plaintext_modulus::Integer)
```
For argument names, we use Julia conventions, i.e., `snake_case`